integration-tests/templates/rn71-template/android/build.gradle

```diff
@@ -1,11 +1,15 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
+// NOTE
+// Embrace won't be able to build/initialize with default values provided by the RN 0.71.x template.
```

Comment:
could add this same comment to rn72->75 templates as well since those were also modified, there's less changes on the later ones so maybe don't have to call out the specific updates